TOPSPIN/Phase_Correction/4D.md

```diff
@@ -1,36 +1,286 @@
+
+**Example Data:** `Raw_Spectra/Carbonic_Anhydrase_2/hCA2-wt_850/6`
+
+This assumes you have measured the **F3-F4 (N-HN) plane** with the same pulse sequence before recording the whole 4D spectrum.
+
+1. **Prepare the Experiment:**
+   - Use the **F3-F4 plane** for phase correction.
+   - Run `wrpa` to create a copy of the whole experiment; name it `61`.
+
+2. **Transform the 2D Plane:**
+   - Open the raw 2D plane in experiment `61` and set all phase values to zero with the following command: 
+```
+4 PHC0 0
+4 PHC1 0
+3 PHC0 0
+3 PHC1 0
+2 PHC0 0
+2 PHC1 0
+1 PHC0 0
+1 PHC1 0
+```
+   - Run `xfb` for FT transform.
+
+![Whole 34 Plane](images/whole_34_plane.png)
+
+   - Select the region of interest (RIO) to zoom in.
+
+![whole 34 plane](images/zoommed_34_plane.png)
+
+   - Right-click anywhere on the spectrum and then select `Save Display Region To -> Parameters STSR/STSI (used by strip ft)`
+
+![whole 34 plane](images/STSR_zoomed_34_plane.png)
+
+After this, the boundaries of the ROI will appear in `STSR` and `STSI` parameters under the `PROCPARS` tab.
+
+   - Right-click again anywhere on the spectrum and this time select `Save Display Region To -> 
+Parameters ABSF1/2 (e.g. used by 'absf, apkf')`. This is to ensure that the baseline correction takes place downfield (to the left)
+of the water line (`~4.5 ppm` in `HN`) to avoid filling the spectrum with noise. Then issue the following commands:
+
+```
+1 ABSF1 1000
+1 ABSF2 -1000
+1 BC_mod no
+2 BC_mod qpol
+2 SSB 2.2
+2 SI SI*2
+1 SI SI*4
+```
+
+- These commands have the following effects:
+  - Leave the default `ABSF1` and `ABSF2` values in the `N` dimension.
+  - Select the `qpol` polynomial function since `qfil` suppresses water very aggressively and is not recommended for the `N-HN` spectrum.
+  - Set `BC_mod` to `no` in the `N` dimension due to its sensitivity to the polynomial.
+  - Set the `SSB` of the direct dimension to `2.2` for further resolution.
+  - Increase the `SI` size of the direct dimension by 2 times to enhance resolution.
+  - Increase the indirect dimension size by four times (zero filling). Ensure that **all SI values are powers of 2** (e.g., 1024, 2048, etc.).
+
+- Execute `xfb` followed by `abs1` and `abs2` for baseline correction. This will remove the irrelevant HN shift region,
+including the water line, while keeping only the ROI. It will also result in sharper and better-resolved peaks due to 
+zero filling and increased SSB.
+
+3. **Prepare for Manual Phase Correction in the F2 dimension:**
+  - Enter `.ph` for manual phase correction.
+  - Look for peaks that have an anti-phase component (signals of dispersion character) in the F2 dimension.
+  - Place the cursor between the centers of these two peaks. Ensure the cursor is along the line connecting them. 
+  - Right-click and select **"Add"**.
+
+![Add Peak](images/add_peak.png)
+
+   - Try to select peaks that cover the span of the whole spectrum; 4 or 5 peaks should suffice.
+
```

Comment:
May add here: 

    - Note: it is useful to note the approximate F2 chemical shifts of the selected peaks. Knowing the frequency of the target peaks will help distinguishing them from the slices of other, non-target, peaks which may also appear in the 1D traces.